src/protocols/uniswapV2.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I would assume you have a way to ensure this is the ABI coming from them?

Reply:
I took it from the etherscan page